runtime/shared_common/CacheMap.cpp

```diff
@@ -984,6 +984,8 @@ SH_CacheMap::updateROMSegmentListForCache(J9VMThread* currentThread, SH_Composit
 			currentROMClass += currentROMSize;
 		}
 		currentSegment->heapAlloc = cacheAlloc;
+		VM_AtomicSupport::writeBarrier();
```

Comment:
Pls move the above variables currentSegSize and maxSegmentSize into the inner scope.

Reply:
Fixed